src/compiler/utilities.ts

```diff
@@ -3334,6 +3340,10 @@ namespace ts {
                 return 14;
             case SyntaxKind.AsteriskAsteriskToken:
                 return 15;
+            case SyntaxKind.BarBarEqualsToken:

```

Comment:
This section doesn't seem to cover any of the assignment or compound assignment operators. I'd get clarity from whoever originally wrote this function whether this should be here, because I think these 3 cases don't belong here.

Reply:
Yeah, it seems to be caught by `getOperatorPrecedence` above.